scripts/release.ts

```diff
@@ -66,6 +67,64 @@ async function undoLastCommit() {
   console.log(c.green('Publish commit deleted.'));
 }
 
+const newDesignSystemVersion = (): string => {
+  const newDesignSystemVersionJSON = JSON.parse(sh(`npm version -w @cmsgov/design-system --json`));
+  return newDesignSystemVersionJSON['@cmsgov/design-system'];
+};
+
+const updateDSVersion = (json: JSON | any): JSON => {
+  const dsKey = '@cmsgov/design-system';
+  json.dependencies[dsKey] = newDesignSystemVersion();
+  return json;
+};
+
+/*
+ * Utilize `npm ls` to get all members of our design system workspace in JSON format.
+ * This allows us to dynamically update our child package.json files later on
+ *  rather than explicitly enumerating our examples and child design systems.
+ * The info we want (member name and package.json file location) are at different levels in the JSON,
+ *  so we have to dig a bit.
+ * We grab the package names by getting all of the keys in the object associated with the 'dependencies' key.
+ * We loop over the names to get the package.json file locations that are in the object associated with the 'resolved' key.
+ */
+const getPackageNamesAndLocations = (): Array<Array<string>> => {
+  const packageJson: any = JSON.parse(sh('npm ls -ws @cmsgov/design-system --json'));
+  const deps = packageJson['dependencies'];
+  const packageNames = Object.keys(deps);
+  return packageNames.map((name: any) => {
+    return [name, deps[name]['resolved']];
+  });
+};
+
+/*
+ * Actually does the updateing of the package.json files.
+ * Gets the file location, reads said file, bumps the version, writes the file.
+ */
+const updatePackageJson = (childPackageToUpdate: Array<string>): void => {
+  // Get our package name and package.json location via destructuring
+  const [packageName, packageLocation] = childPackageToUpdate;
+  /*
+   * The package location is stored relative to the root directory when returned by `npm ls`,
+   * so we have to clean it up a bit so `path` knows where to look.
+   */
+  const cleanedPackageLocation = packageLocation.split('file:')[1].replaceAll('../', '');
```

Comment:
I think this would read better if you handled the package location updates in the `getPackageNamesAndLocations` function. That way the `getPackageNamesAndLocations` function can just focus on the package JSON updates.